content/ledger-objects/directorynode.md

```diff
@@ -0,0 +1,94 @@
+## DirectoryNode
+[[Source]<br>](https://github.com/ripple/rippled/blob/5d2d88209f1732a0f8d592012094e345cbe3e675/src/ripple/protocol/impl/LedgerFormats.cpp#L44 "Source")
+
+The `DirectoryNode` object type provides a list of links to other objects in the ledger's state tree. A single conceptual _Directory_　takes the form of a doubly linked list, with one or more DirectoryNode objects each containing up to 32 [IDs](#tree-format) of other objects. The first object is called the root of the directory, and all objects other than the root object can be added or deleted as necessary.
+
+There are two kinds of Directories:
+
+* **Owner directories** list other objects owned by an account, such as `RippleState` or `Offer` objects.
+* **Offer directories** list the offers available in the distributed exchange. A single Offer directory contains all the offers that have the same exchange rate for the same issuances.
+
+Example Directories:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*Offer Directory*
+
+```json
+{
+    "ExchangeRate": "4F069BA8FF484000",
+    "Flags": 0,
+    "Indexes": [
+        "AD7EAE148287EF12D213A251015F86E6D4BD34B3C4A0A1ED9A17198373F908AD"
+    ],
+    "LedgerEntryType": "DirectoryNode",
+    "RootIndex": "1BBEF97EDE88D40CEE2ADE6FEF121166AFE80D99EBADB01A4F069BA8FF484000",
+    "TakerGetsCurrency": "0000000000000000000000000000000000000000",
+    "TakerGetsIssuer": "0000000000000000000000000000000000000000",
+    "TakerPaysCurrency": "0000000000000000000000004A50590000000000",
+    "TakerPaysIssuer": "5BBC0F22F61D9224A110650CFE21CC0C4BE13098",
+    "index": "1BBEF97EDE88D40CEE2ADE6FEF121166AFE80D99EBADB01A4F069BA8FF484000"
+}
+```
+
+*Owner Directory*
+
+```json
+{
+    "Flags": 0,
+    "Indexes": [
+        "AD7EAE148287EF12D213A251015F86E6D4BD34B3C4A0A1ED9A17198373F908AD",
+        "E83BBB58949A8303DF07172B16FB8EFBA66B9191F3836EC27A4568ED5997BAC5"
+    ],
+    "LedgerEntryType": "DirectoryNode",
+    "Owner": "rpR95n1iFkTqpoy1e878f4Z1pVHVtWKMNQ",
+    "RootIndex": "193C591BF62482468422313F9D3274B5927CA80B4DD3707E42015DD609E39C94",
+    "index": "193C591BF62482468422313F9D3274B5927CA80B4DD3707E42015DD609E39C94"
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+| Name              | JSON Type | [Internal Type][] | Description |
+|-------------------|-----------|---------------|-------------|
+| `LedgerEntryType`   | String    | UInt16    | The value `0x0064`, mapped to the string `DirectoryNode`, indicates that this object is part of a Directory. |
+| `Flags`             | Number    | UInt32    | A bit-map of boolean flags enabled for this directory. Currently, the protocol defines no flags for DirectoryNode objects. |
```

Comment:
I like this `Flags` descr as well. Perhaps a combination of this and the one from check.md can be used to describe `Flags` when no flags will ever be defined for an object? Sorry, now it seems that I'm fixated on flags...LOL. I guess what I'm trying to say is that it would be good to have a comprehensive and consistent descr for when the `Flag` field value will always be set to `0`.